include/mbedtls/pk.h

```diff
@@ -522,24 +476,17 @@ int mbedtls_pk_import_into_psa(const mbedtls_pk_context *pk,
                                mbedtls_svc_key_id_t *key_id);
 
 /**
- * \brief           Create a PK context starting from a key stored in PSA.
+ * \brief           Set up a PK context with the key material from a PSA key
+ *                  pair.
```

Comment:
Not necessarily a key pair, it can be a public key.

Reply:
Duh, it says right below "key pair or public key". What was I thinking.